classes/object_manipulator/pusher.php

```diff
@@ -48,6 +48,13 @@ class pusher extends manipulator {
     private $minimumage;
 
     /**
+     * The maximum upload file size in bytes.
```

Comment:
My bad for leaving them in here but do you mind removing the 'use Aws\S3\Exception\S3Exception;' statments from the puller, pusher, recoverer and deleter?